src/NuProj.ProjectSystem.12/Rules/general.xaml

```diff
@@ -1,33 +1,46 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You've made a lot of whitespace changes to this file which makes it hard to filter out the signal from the noise. I'd prefer if you could adjust your pull request to only include meaningful changes, and save any whitespace changes you want to make for a separate PR so we can more efficiently review your changes.